README.md

```diff
@@ -1,3 +1,38 @@
-# Blackhole.service for DEIS
+# Blackhole service for Deis
 
-This will be a repository for dockerized blackhole service for DEIS.
+A repository for a docker container with simple Blackhole implementation for Deis (CoreOS) platform.
+
+The script bans every failed attempt to login by password on CoreOS instances except ip addresses that are included in `WHITELIST` variable.
```

Comment:
I don't understand it. It will not allow to login with password for ip address that is not listed in `WHITELIST`? or it will ban ip address when it tries to login with a wrong password?